src/assets/config_files/usecases.json

```diff
@@ -626,6 +626,7 @@
           "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/objectives.png"
         },
         "next": "sc_form",
+	"tutorial": "https://object.cscs.ch/v1/AUTH_c0a333ecf7c045809321ce9d9ecdfdea/web-resources-bsp/bsp-video-tutorials/BuildYourOwnSingleHippocampalCellModel/index.html",
```

Comment:
Hi Carmen, Could you please indent this new element the same way that you have the other so it's more readable ?